core/src/main/java/android/support/v4/app/SafeJobIntentService.java

```diff
@@ -1,12 +1,12 @@
-package android.support.v4.app;
+package androidx.core.app;
```

Comment:
Should the file, `core/src/main/java/android/support/v4/app/SafeJobIntentService.java`, be moved to `androidx/core/app` directory?